### DISCUSSION TITLE
replace figures in revealjs

### DISCUSSION THREAD

**Discussion by vanAmsterdam** [quarto-dev/quarto-cli#9384] [Q&A] [answered]
### Description

I use sequences of related figures for teaching, e.g. same figure with different highlighting / uncovering different parts
I'd like to use a sequence of them in a single slide. In beamer you can do:

```
\includegraphics<1>{fig2.pdf}
\includegraphics<2>{fig2.pdf}
```

many times text / equations would change at the same time
would it be doable to copy this behavior in quarto's revealjs format?

If found this post about it in revealjs: https://stackoverflow.com/questions/23608762/replace-image-in-reveal-js

**Comment by mcanouil** [comment, marked as answer]
It's already possible use fragment/stack, see <https://quarto.org/docs/presentations/revealjs/advanced.html>.
Beyond that, you can look at [Reveal.js documentation](https://revealjs.com/) and/or use any HTML/(S)CSS based tricks.

See for example <https://github.com/quarto-dev/quarto-cli/discussions/9383#discussioncomment-9127900>.

**Comment by vanAmsterdam** [comment]
wow how did I miss this; thanks for the pointer!
